week2/project/ecommerce/src/components/Categories.jsx

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable react/prop-types */
+export default function Categories({
+    categories,
+    selectedCategory,
+    onCategorySelect
+}) {
+    return (
+        <section>
+            <ul className="categories">
+                {categories.map((category, index) => (
+                    <li
+                        key={index}
+                        className={`categories--item ${
+                            selectedCategory === category
+                                ? 'categories--item-selected'
+                                : ''
+                        }`}

```

Comment:
```suggestion
                        className={`categories--item ${
                            selectedCategory === category
                                &&  'categories--item-selected'
                        }`}

```

Nice-to-have: Shorter

Reply:
Done